code/modules/mob/living/silicon/ai/icons.dm

```diff
@@ -163,14 +165,87 @@ var/list/datum/ai_icon/ai_icons
 /datum/ai_icon/trapped
 	name = "Trapped"
 	alive_icon = "ai-hades"
-	dead_icon = "ai-hades_dead"
+	dead_icon = "ai-hades-crash"
 
-/datum/ai_icon/triumvirate_static
+/datum/ai_icon/triumvirate
 	name = "Triumvirate"
 	alive_icon = "ai-triumvirate"
 	alive_light = "#020B2B"
 
 /datum/ai_icon/triumvirate_static
 	name = "Triumvirate Static"
-	alive_icon = "ai-static"
+	alive_icon = "ai-triumvirate-malf"
 	alive_light = "#020B2B"
+
+//Eros Research Platform Ports

```

Comment:
Any idea where these sprites were originally acquired by Eros? Specifically Boxfort, and any including and below House. I'm not distinctly concerned for references, because the base AI sprites are references, simply concerned if it's kosher to use them.

Reply:
I also made one of the sprites included, which is kind of a blast-from-the-past kind of deal. But, alright.

Reply:
I recall the human-attached-to-the-core sprites being from Bay (I think it was Ravensdale's sprite).
The 'gentoo' core animation is actually mine from /tg/ (surprised to see it again) which references https://www.youtube.com/watch?v=VjGSMUep6_4.

Reply:
The Strogg-looking "half a corpse hanging from the core" ones with a roboarm is indeed mine (regular and the "dead" version), as are the Triumvirate sprites. Made them back on Exodus!Bay.